vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -1518,4 +1522,108 @@ winrt::Microsoft::ReactNative::ComponentView WindowsTextInputComponentView::Crea
   return winrt::make<WindowsTextInputComponentView>(compContext, tag, reactContext);
 }
 
+void WindowsTextInputComponentView::autoCapitalizeOnUpdateProps(const std::string &capitalizationType) noexcept {
+  /*
+    Possible values are:
+     Characters - All characters.
```

Comment:
For characters at least, we should just use EM_SETEDITSTYLE  and SES_UPPERCASE - to have RichEdit do this properly.